cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,35 @@
 /// <reference types='cypress' />
 
-describe('', () => {
+import HomeAndCataloguePageObject from '../support/pages/homeСatalogue.pageObject';
+import ContactFormPageObject from '../support/pages/contactForm.pageObject';
+
+const homeAndCatalogue = new HomeAndCataloguePageObject();
+const contactForm = new ContactFormPageObject();
+
+
+describe('ergeg', () => {
```

Comment:
The describe block has an unclear name. Rename it to something descriptive related to the test suite being run.